TickSpec/FeatureParser.fs

```diff
@@ -91,12 +91,20 @@ let parseFeature (lines:string[]) =
         let bullets, table, doc =
             match step.Item with
             | Some (BulletsItem b) ->
-                b |> List.toArray |> Some, None, None
+                b
+                |> List.map (replace combination)
```

Comment:
Seq.map >> Seq.toArray

Reply:
Not sure I understand this one, b is string list, I am just List.mapping before the List.toArray which was already there ..

Reply:
It doesnt need to be a cons list with lots of allocations - just because it started as a List does not mean it has to stay one

Reply:
I have pushed sth - is this is what you expect?

Reply:
Yes, looks good!